src/index.js

```diff
@@ -39,6 +40,10 @@ export const MODULE_TRANSFORMATIONS = {
  * @return {Boolean}  Whether or not the transformation is required
  */
 export const isPluginRequired = (supportedEnvironments, plugin) => {
+  if (supportedEnvironments.browsers) {
```

Comment:
@hzoo It's just for usage from a separate module. When options are not handled yet.